### PR TITLE
Java. Add DieselPublisherManagerService

### PR DESCRIPTION
**Pull request by aidar-aibekov**
* rename Preparer to Builder

### CODE REVIEW

File: share/src/main/java/io/zensoft/share/service/diesel/DieselPublisherManagerService.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        this.vacancyResponseConverterService = vacancyResponseConverterService;
+        this.vacancyModelService = vacancyModelService;
+        this.vacancyResponseModelService = vacancyResponseModelService;
+        this.dieselPublisherService = dieselPublisherService;
+        this.vacancyRetrieverService = vacancyRetrieverService;
+        this.vacancyResponseSenderService = vacancyResponseSenderService;
+    }
+
+    @Override
+    @Transactional
```
Comment (derkenbaev) [inline share/src/main/java/io/zensoft/share/service/diesel/DieselPublisherManagerService.java]: Does it mean that if an exception is thrown during publishing, the will be no saving to db and responding?

File: share/src/main/java/io/zensoft/share/service/diesel/body/PublicationVacancyContentBuilder.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 @Data
 @Component
-public class PublicationVacancyContentPreparer {
+public class PublicationVacancyContentBuilder {
```
Comment (PlusRT) [inline share/src/main/java/io/zensoft/share/service/diesel/body/PublicationVacancyContentBuilder.java]: I think this class is service

File: share/src/main/java/io/zensoft/share/service/diesel/DieselPublisherManagerService.java (hunk 1 above)
Comment (aidar-aibekov) [reply share/src/main/java/io/zensoft/share/service/diesel/DieselPublisherManagerService.java]: Yes. Do you suggest to delete @transactional?

Comment (derkenbaev) [reply share/src/main/java/io/zensoft/share/service/diesel/DieselPublisherManagerService.java]: I think in this case we should save to db with failed status and respond. But it is not a problem if you handle exceptions. The question is should we be able to respond, save to db or publish, if other step is failed?

**Pull request merged**